test/katex-spec.js

```diff
@@ -1116,6 +1116,31 @@ describe("A markup generator", function() {
     });
 });
 
+describe("A parse tree generator", function() {
+    it("generates a tree", function() {
+        var tree = katex.__parse("\\sigma^2");
+        expect(JSON.stringify(tree)).toEqual(JSON.stringify([
```

Comment:
No, never mind, `toEqual` does do deep compare: [jasmine's deep `eq` function](https://github.com/jasmine/jasmine/blob/33641578e6f7d90be6155ea6dcb262b11a326fea/src/core/matchers/matchersUtil.js#L53). The only thing I can think of is that `"sub": undefined` should be `"sub": null`?